examples/coap_client/coap_client.c

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (c) 2015 Cesanta Software Limited
+ * All rights reserved
+ *
+ * This program sends CoAP CON-message to server (coap.me by default) 
+ * and waits for answer.
+ */
+
+#include "fossa.h"
+
+static int g_time_to_exit = 0;
+static char* g_default_address = "udp://coap.me:5683";
+
+static void coap_handler(struct ns_connection *nc, int ev, void *p) {
+  switch (ev) {
+    case NS_CONNECT: {
+      struct ns_coap_message cm;
+      uint32_t res;
+
+      memset(&cm, 0, sizeof(cm));
+      cm.msg_id = 1;
+      cm.msg_type = NS_COAP_MSG_CON;
+      fprintf(stdout, "Sending CON...\n");
```

Comment:
On which platform?

AFAIK both behave the same, and flush the output on newline.